### DISCUSSION TITLE
Stability and load of python

### DISCUSSION THREAD

**Discussion by markusthiel** [photobooth-app/photobooth-app#190] [General]
Sometimes the app seems to be a bit unstable. I have noticed that it reloads now and then. Or after pressing the trigger, the screen turns white and gets stuck there, but then eventually reloads and goes back to the home screen. However, a photo is not taken then.

And under the load, it always shows me 100% from Python on my Raspberry Pi 5.

**Comment by mgineer85** [comment]
Hi, 
please add the latest logs here and it would be helpful if you can describe your setup a bit more in detail (backend, os (64bit required)).

On a Pi5 the load is a bit higher than for the Pi4 (because it lacks hardware acceleration) but it should be below 50-60% usually depending on the camera setup.

If you switch to the virtual camera backend, is the load reduced to normal?

I think the stability issue is because of the high load.   You have a cooler applied to the Pi5?

**Reply by mgineer85** [reply]
Please also do not misinterpret: The PIs have 4 cores, so they could handle actually 400% load until the cpu is totally busy.

**Reply by mgineer85** [reply]
This is how it is looking normally in htop:
![image](https://github.com/photobooth-app/photobooth-app/assets/3868450/7f0a2ad0-46de-4af3-9813-1287e9846bca)

**Reply by mgineer85** [reply]
I updated my system to latest version and also system upgraded - still everything normal on my side. So I need more information to find the issue in your setup.
